nemo_skills/training/data_preparation_utils/filters.py

```diff
@@ -148,6 +148,38 @@ def process_dataset_entry(self, data_entry) -> List:
         return [DataEntry(data=data_entry, metrics=dict(num_removed=0))]
 
 
+class RemoveLenOutlierProblems(BaseFilter):
```

Comment:
Isn't it the same as `RemoveLenOutlierSolutions`? If so, let's just name it more generally and then use as it is by changing the `problem_key` (which we should also name more generally)

Reply:
Replaced these with a general function